### PR TITLE
ESLint: Quick clean up

### PR DESCRIPTION
**Pull request by Pezmc**
## Description

Some quick cleanup of low hanging ESLint fruit between releases before new branches are opened (should be merged after release).

### CODE REVIEW

**Comment by Steve-Mcl** [comment]
@Pezmc , ping me when you are back at work please. would like to pick a suitable time to review and merge.  I am certain any time if fine want to check with you first.

**Comment by Pezmc** [comment]
@Steve-Mcl It's a shame this couldn't go in while I was out, I had hoped to get it merged before everyone created more branches to help avoid merge conflicts, but yes, will ping you when I see you're online.

**Comment by knolleary** [comment]
@Steve-Mcl too late for 1.7 now, but what was blocking getting this merged?

**Comment by Steve-Mcl** [comment]
Nothing much Nick. a couple of questions I had at the time but more than anything, I wanted to check and coordinate the work with Pez to announce in the slack channel a suitable merge time so as not to disrupt too much WIP. Just turned out the timing of vacations caused weeks of delayed responses and it fell off the radar.

**Comment by Pezmc** [comment]
I propose as soon as 1.7 is released, I rebase this onto main and fix up any new eslint failures, then we merge asap before too many new branches are opened.

**Comment by Steve-Mcl** [comment]
@Pezmc agreed. To ensure this doesnt slip through the net again, lets make sure we alert each other when we are ready to rock n roll this.

**Comment by Pezmc** [comment]
Rebased onto main, this PR is good to go out as soon as it's approved @knolleary @Steve-Mcl, ideally that's by end of day today.

**Comment by Steve-Mcl** [comment]
Good stuff. Gonna do it within the next hour Pez

File: .eslintrc [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "parserOptions": {
         "ecmaVersion": 2022
     },
-    "ignorePatterns": ["frontend/dist/", "var/"],
+    "ignorePatterns": ["frontend/dist/", "var/", "*.svg", "*.xml"],
     "plugins": ["promise"],
     "settings": {
         "import/core-modules": [ "lottie-web-vue", "@heroicons/vue" ],
```
[the remark this answers is not in the document]
Comment (knolleary) [reply .eslintrc:18]: @Steve-Mcl can you just merge that suggestion and get this merged please?

[the remark this answers is not in the document]
Comment (knolleary) [reply .eslintrc:18]: Actually - no - that isn't a dangling comma at all

[the remark this answers is not in the document]
Comment (knolleary) [reply .eslintrc:18]: ignore me - it is.

**Review by knolleary — APPROVED** [review]

**Pull request merged**